[DOCS] 7.9 Release Notes (#74306)
elastic/kibana: docs/CHANGELOG.asciidoc

* Triggers Lazy-load alert {kibana-pull}68806[#68806]
* Changes to duration formatting {kibana-pull}69039[#69039]
* Adds callout to inform users of high cardinality in unique transaction names {kibana-pull}69112[#69112]
* Creates API to return data to be used on the Overview page {kibana-pull}69137[#69137]
* Fixes confusing request/minute viz {kibana-pull}69143[#69143]
* Adds decimals only for numbers below 10 {kibana-pull}69334[#69334]
* Adds support for dark mode {kibana-pull}69362[#69362]
* Chart breakdowns {kibana-pull}69420[#69420]
* Quotes trace id to ensure a word is searched (#69500) {kibana-pull}69504[#69504]
* Adds error rates to Service Map popovers {kibana-pull}69520[#69520]
* Resubmits initial version {kibana-pull}69531[#69531]
* Adds 'Anomaly detection' settings page to create ML jobs per environment {kibana-pull}70560[#70560]
* Service maps anomaly detection integration by environment {kibana-pull}70932[#70932]
* Anomaly detection setup link with alert if job doesn't exist {kibana-pull}71229[#71229]
* Respects default time range defined in Kibana Advanced Settings {kibana-pull}71464[#71464]
* Uses HDR for percentiles {kibana-pull}64758[#64758]
Dashboard::

> * Adds Lens drilldowns {kibana-pull}65675[#65675]

Reviewer: Reword "Enables drilldowns to Lens visualizations"

> * Improves duplicate panel positioning {kibana-pull}67461[#67461]

Reviewer: Improves positioning of cloned panels on Dashboard

Discover::
* Multi-line kql bar {kibana-pull}70140[#70140]
Ingest Management::
* Adds support for datastream to each template {kibana-pull}66367[#66367]
* Improves server-side error handling {kibana-pull}67278[#67278]
* Adds ability to copy an agent config {kibana-pull}68737[#68737]
* Adds enroll agent action to config action menu {kibana-pull}68840[#68840]
* Adds ability to sort to agent configs and package configs {kibana-pull}70676[#70676]
Kibana UI::

> * Adds link for Cloud deployment settings {kibana-pull}66486[#66486]

Reviewer: I wonder if we should remove this - seems like it would only be applicable on the Cloud and those users will see it anyway?

* Adds back context variables with descriptions {kibana-pull}67487[#67487]
* Adds overrides to Snapshot API to support alert previews {kibana-pull}68125[#68125]
* Enhances Inventory View Tooltips {kibana-pull}69757[#69757]
* UX improvements for saved views {kibana-pull}69910[#69910]
* Register function for Observability homepage {kibana-pull}70529[#70529]
* Adds framework for recovery messaging to metric threshold alerts (non-functional) {kibana-pull}65339[#65339]
* Adds preview feature for metric threshold alerts {kibana-pull}67684[#67684]
* Adds inventory alert preview {kibana-pull}68909[#68909]
* Prefills alerts from the global dropdown {kibana-pull}68967[#68967]
* Adds context.reason and alertOnNoData to Inventory alerts {kibana-pull}70260[#70260]
Monitoring::
* Collects number of visualization saved in the past 7, 30 and 90 days {kibana-pull}67865[#67865]
* Out of the box alerting {kibana-pull}68805[#68805]
Operations::
* Node options from cfg file for production {kibana-pull}62468[#62468]
* Creates Linux aarch64 package {kibana-pull}69165[#69165]
Platform::
* Switches to core application service {kibana-pull}63443[#63443]

> * Lens editor auto refresh {kibana-pull}65868[#65868]

Reviewer 2: Move to "Dashboard" section

> * Adds simple variables to workpads {kibana-pull}66139[#66139]

Reviewer 2: Move to Canvas section

* Adds docLinks to CoreSetup {kibana-pull}66631[#66631]
* New Enterprise Search Kibana plugin {kibana-pull}66922[#66922]

> * Workplace Search in Kibana MVP {kibana-pull}70979[#70979]

Reviewer 2: Not sure if we have one, but these should probably be in a Enterprise Search section?

Author: @rayafratkina are we in a position to add an Enterprise Search section to the Kibana Release Notes?

* <<release-notes-7.0.0-alpha1>>

--
[[release-notes-7.9.0]]
== {kib} 7.9.0
coming[7.9.0]

//See also <<breaking-changes-7.9,breaking changes in 7.9>>.

Operations::
* Moves kibana-keystore from data/ to config/ {kibana-pull}57856[#57856]

[float]
[[enhancement-7.9.0]]
=== Enhancements

> Alerting::

Reviewer 3: All these items under **Alerting** can move to the **Metrics** as they are specific to metric alerts.

> * Adds framework for recovery messaging to metric threshold alerts (non-functional) {kibana-pull}65339[#65339]

Reviewer 4: The shared attribute is {kib-pull} so these aren't resolving.  I think you need to either change them all, or define {kibana-pull} at the top of this page.
```suggestion
* Adds framework for recovery messaging to metric threshold alerts (non-functional) {kib-pull}65339[#65339]
```

Author: Resolved in #1933

* Adds preview feature for metric threshold alerts {kibana-pull}67684[#67684]
* Adds inventory alert preview {kibana-pull}68909[#68909]
* Prefills alerts from the global dropdown {kibana-pull}68967[#68967]
* Adds context.reason and alertOnNoData to Inventory alerts {kibana-pull}70260[#70260]
APM::
* Shows `trace.id`, `transaction.id` and/or `error.id` in metadata table {kibana-pull}66376[#66376]

> * Adds error rate chart to Errors overview and detail views {kibana-pull}67327[#67327]

Reviewer 5: We changed the location of the error rate, could you please update it?
```suggestion
* Adds error rate chart to Transaction overview and detail views {kibana-pull}67327[#67327]
```

* Allows histogram fields in average and sum aggregations {kibana-pull}66891[#66891]
* Changes the error message on TSVB in order to be more user friendly {kibana-pull}67090[#67090]
* Allows the user to change the tooltip mode {kibana-pull}67775[#67775]
* Updates vega version {kibana-pull}68639[#68639]
* Adds support for histogram type {kibana-pull}68837[#68837]
* Adds visualization by value to dashboard {kibana-pull}69898[#69898]
Logs::
* Validates ML job setup time ranges {kibana-pull}66426[#66426]
* [Alerting] "Group by" functionality {kibana-pull}68250[#68250]
* ML log integration splash screen {kibana-pull}69288[#69288]
* Actions menu in log entry categorization page {kibana-pull}69567[#69567]
* Adds index names for the new indexing strategy {kibana-pull}70245[#70245]
* Adds category anomalies to anomalies page {kibana-pull}70982[#70982]
* Anomalies page dataset filtering {kibana-pull}71110[#71110]
* Shows log analysis ML jobs in a list {kibana-pull}71132[#71132]
Machine Learning::
* Search should have a categorical option for job type {kibana-pull}65770[#65770]
* Adds linking to dataframe from job management tab {kibana-pull}65778[#65778]

> * Adds optional ability to delete target index and index pattern when deleting DFA job {kibana-pull}66934[#66934]

Reviewer 4: ```suggestion
* Adds optional ability to delete target index and index pattern when deleting {dfanalytics-job} {kib-pull}66934[#66934]
```

> * DF Analytics: Creation wizard part 1 {kibana-pull}67564[#67564]

Reviewer 4: ```suggestion
* {dfanalytics-cap}: Creation wizard part 1 {kib-pull}67564[#67564]
```

* Extends population preview chart to show actual and typical value {kibana-pull}67569[#67569]

> * Adds minor refresh button to DFA and AD Job Messages tabs {kibana-pull}67750[#67750]

Reviewer 4: ```suggestion
* Adds minor refresh button to {dfanalytics} and {anomaly-detect} Job Messages tabs {kib-pull}67750[#67750]
```

* Keeps the edit rule flyout open if there are multiple rules {kibana-pull}68174[#68174]
* Model snapshot management {kibana-pull}68182[#68182]

> * DF Analytics: Creation wizard part 2 {kibana-pull}68462[#68462]

Reviewer 6: ```suggestion
* {dfanalytics-cap}: Creation wizard part 2 {kib-pull}68462[#68462]
```

* Removes sub navigation menu from the Anomaly Detection pages {kibana-pull}68663[#68663]

> * Adds Anomaly Swimlane Embeddable to the dashboard from the Anomaly Explorer page {kibana-pull}68784[#68784]

Reviewer 4: ```suggestion
* Adds anomaly swim lane embeddable to the dashboard from the Anomaly Explorer page {kib-pull}68784[#68784]
```

* Allows editing of model_snapshot_retention_days {kibana-pull}69003[#69003]
* Anomaly Explorer swim lane pagination {kibana-pull}70063[#70063]
* Anomaly Detection: Annotations enhancements {kibana-pull}70198[#70198]
* Updates APM Module to Work with Service Maps {kibana-pull}70361[#70361]
* Changes all calls to ML endpoints to use internal user {kibana-pull}70487[#70487]

> * DF Analytics: add ability to edit job for fields supported by API {kibana-pull}70489[#70489]

Reviewer 6: ```suggestion
* {dfanalytics-cap}: add ability to edit job for fields supported by API {kib-pull}70489[#70489]
```

> * DF Analytics: adds prompt for destination index pattern creation {kibana-pull}70651[#70651]

Reviewer 6: ```suggestion
* {dfanalytics-cap}: adds prompt for destination index pattern creation {kib-pull}70651[#70651]
```

* Adds back context variables with descriptions {kibana-pull}67487[#67487]
* Adds overrides to Snapshot API to support alert previews {kibana-pull}68125[#68125]
* Enhances Inventory View Tooltips {kibana-pull}69757[#69757]
* UX improvements for saved views {kibana-pull}69910[#69910]
* Register function for Observability homepage {kibana-pull}70529[#70529]
Monitoring::
* Collects number of visualization saved in the past 7, 30 and 90 days {kibana-pull}67865[#67865]
* Out of the box alerting {kibana-pull}68805[#68805]
Operations::
* Node options from cfg file for production {kibana-pull}62468[#62468]
* Creates Linux aarch64 package {kibana-pull}69165[#69165]
Platform::
* Switches to core application service {kibana-pull}63443[#63443]
* Lens editor auto refresh {kibana-pull}65868[#65868]
* Adds simple variables to workpads {kibana-pull}66139[#66139]
* Adds docLinks to CoreSetup {kibana-pull}66631[#66631]
* New Enterprise Search Kibana plugin {kibana-pull}66922[#66922]
* Workplace Search in Kibana MVP {kibana-pull}70979[#70979]
Querying & Filtering::
* Resolves range date filter bugs and improve usability {kibana-pull}71298[#71298]
Security::

> * Searches across spaces {kibana-pull}67644[#67644]

Reviewer 7: ```suggestion
* Allow saved objects to be searched across multiple spaces {kibana-pull}67644[#67644]
```

* [Exceptions] - Updates exception hooks and viewer {kibana-pull}73588[#73588]
* [Exceptions] - Update how nested entries are displayed in exceptions viewer {kibana-pull}73745[#73745]
* [Exceptions] Adds autocomplete workaround for .text fields {kibana-pull}73761[#73761]
* [Endpoint][Exceptions] Only write manifest to policy when there are changes {kibana-pull}72000[#72000]
* [Exceptions] Use semantic version for manifest version + Scaling Tweaks {kibana-pull}73388[#73388]
* Resolver in timeline {kibana-pull}69728[#69728]
Uptime::
* Uses ML Capabilities API to determine license type {kibana-pull}66921[#66921]
* Landing page for Observability {kibana-pull}67467[#67467]
* Monitor availability reporting {kibana-pull}67790[#67790]
* Creates "Add data" links in all Observability app headers {kibana-pull}69016[#69016]
* Observability overview page {kibana-pull}69141[#69141]
* Availability alert {kibana-pull}70284[#70284]
* Duration Anomaly Alert {kibana-pull}71208[#71208]

[float]
[[bug-v7.9.0]]
=== Bug fixes
Alerting::
* Adds lazy loading of alerting UI components {kibana-pull}65060[#65060]

> * Migrates the old `alerting` consumer to be `alerts` {kibana-pull}69982[#69982]

Reviewer 3: This one can be removed as the PR got reverted (https://github.com/elastic/kibana/pull/73420)

* Displaya Too Many Buckets error when previewing Inventory Alerts {kibana-pull}70508[#70508]

> * Round metric threshold time buckets to nearest unit {kibana-pull}71172[#71172]

Reviewer 3: These two can move to **Metrics** section.

* Removes no longer required div wrapper around ValidatedDualRange {kibana-pull}70188[#70188]
Metrics::
* Fixes asynchronicity and error handling in Snapshot API {kibana-pull}70503[#70503]
* Fixes evaluating rate-aggregated alerts when there's no normalized value {kibana-pull}73545[#73545]
Monitoring::
* Adds a case for Alerting if security/ssl is disabled {kibana-pull}71846[#71846]
* Checks for security feature first when entering setup mode {kibana-pull}73821[#73821]
Operations::
* Manages pid folder {kibana-pull}68262[#68262]
Platform::
* Sets wrap to the errorLink so it doesn't go outside of box {kibana-pull}67797[#67797]
* Fixes plugin lifecycle log to only include server plugins {kibana-pull}68686[#68686]
Querying & Filtering::
* When using KQL or the filter bar, if a request is issued to Elasticsearch for suggestions for a value for a specific field, the request 
will be cancelled when navigating away as well as when the input is updated (and a new request is issued {kibana-pull}69769[#69769]
Reporting::
* Fixes a bug where the Kibana server could crash if the Reporting server-side headless browser crashes {kibana-pull}71989[#71989]

> Security::
> * Clones role button going to edit page {kibana-pull}70768[#70768]

Reviewer 7: ```suggestion
```

Sorry for the mis-label. This was a bug caught prior to shipping, and shouldn't be included as part of the release notes
